packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx

```diff
@@ -320,3 +400,8 @@ type SnapshotUtilizationCardProps = {
   clusters?: string[];
   CustomUtilizationSummary?: React.FC<CustomUtilizationSummaryProps>;
 };
+
+type UtilizationCardProps = {
+  clusterName: string;
+  peerClusters: string[];
```

Comment:
```suggestion
    peerClusters: string[];
```

use only peerClusters

Reply:
We need the selected cluster name for the Replication throughput ...

Reply:
you can pass the clusterName inside peerClusters array and fetch from it